bundles/org.openhab.binding.edf/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="edf" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>EDF Binding</name>
+	<description>This is the binding for EDF.</description>
```

Comment:
... for EDF prices
Tariff or price, I don't know what is the good word in English.

Reply:
I've used Tariff accross the binding, I'll stay with it.